apps/analytics/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+
+class ParticipantTeamCount(object):
```

Comment:
Can we please keep this class in `participant` app since this is a part of the participant app? We might use this at other places as well in future.